Description: Fix format problems in manpages
 This patch fixes minor format problems in manpages that lintian
 complains about.
 .
 It can be removed when 1.3.4 is released.
Forwarded: https://svn.open-mpi.org/trac/ompi/ticket/1973
Author: Manuel Prinz <manuel@debian.org>
Last-Update: 2009-10-13

--- a/orte/tools/orterun/orterun.1in
+++ b/orte/tools/orterun/orterun.1in
@@ -308,7 +308,7 @@
 .
 .
 .TP
-.B -rf \f |--rankfile <arg0>\fP.
+.B -rf\fR,\fP --rankfile <arg0>
 Provide a rankfile file.
 .
 .
@@ -709,7 +709,7 @@
  #Connection        From node   To node:weight   To node:weight
  #declaration       (declared   (declared        (declared
  #(Reserve word)    above)      above)           above)
- #========================================================================
+ #=============================================================
  BRANCH             mem0        socket0:0
  BRANCH             mem3        socket3:0
  #
